core/src/test/scala/kanaloa/reactive/dispatcher/IntegrationTests.scala

```diff
@@ -259,7 +259,7 @@ class AutoScalingWithPullingIntegration extends IntegrationSpec {
       var lastPoolSize = 0
       import system.dispatcher
 
-      fishForMessage(duration * 2) {
+      fishForMessage(duration * 3) {
```

Comment:
I am not sure we should've changed this. It might be an environmental issue. If it's not then the system became slower, which we should find out why. can you try change it back and test one more time?